portal-api/src/v1/api.js

```diff
@@ -270,7 +270,7 @@ const saveUtilisationReport = async (reportData, reportPeriod, user, fileInfo) =
 /**
  * @typedef {Object} GetUtilisationReportsOptions
  * @property {import('../types/utilisation-reports').ReportPeriod} [reportPeriod] - a report period to filter reports by
- * @property {import('../types/utilisation-reports').UtilisationReportReconciliationStatus[]} [reportStatuses] - list of report statuses to filter reports by
+ * @property {('true' | 'false')} [excludeNotUploaded] - whether or not to exclude reports which have not been uploaded
```

Comment:
Any reason not to use a `boolean` here?

Reply:
I suppose not. I think my reasoning was that whenever we send it off in a request, it will become the `string` `'true'` or `'false'`, but seeing as this will be used to construct a request there is no good reason to not use the actual `boolean` values

Reply:
Yeah, string values of `'true'` and `'false'` is a limitation of query params, but we shouldn't use the stringified values anywhere other than where we first extract the values from `req.query`.